geode-connectors/src/distributedTest/java/org/apache/geode/connectors/jdbc/internal/cli/DestroyMappingCommandDunitTest.java

```diff
@@ -66,36 +71,119 @@ public void before() throws Exception {
 
     gfsh.connectAndVerify(locator);
 
-    gfsh.executeAndAssertThat("create region --name=" + REGION_NAME + " --type=REPLICATE")
+    gfsh.executeAndAssertThat("create region --name=" + REGION_NAME + " --type=PARTITION")
         .statusIsSuccess();
+  }
 
+  private void setupAsyncMapping() {
     CommandStringBuilder csb = new CommandStringBuilder(CREATE_MAPPING);
     csb.addOption(CREATE_MAPPING__REGION_NAME, REGION_NAME);
-    csb.addOption(CREATE_MAPPING__DATA_SOURCE_NAME, "connection");
-    csb.addOption(CREATE_MAPPING__TABLE_NAME, "myTable");
+    csb.addOption(CREATE_MAPPING__DATA_SOURCE_NAME, "myDataSource");
     csb.addOption(CREATE_MAPPING__PDX_NAME, "myPdxClass");
 
     gfsh.executeAndAssertThat(csb.toString()).statusIsSuccess();
   }
 
+  private void setupSynchronousMapping() {
+    CommandStringBuilder csb = new CommandStringBuilder(CREATE_MAPPING);
+    csb.addOption(CREATE_MAPPING__REGION_NAME, REGION_NAME);
+    csb.addOption(CREATE_MAPPING__DATA_SOURCE_NAME, "myDataSource");
+    csb.addOption(CREATE_MAPPING__PDX_NAME, "myPdxClass");
+    csb.addOption(CREATE_MAPPING__SYNCHRONOUS_NAME, "true");
+
+    gfsh.executeAndAssertThat(csb.toString()).statusIsSuccess();
+  }
+
+  @Test
+  public void destroysAsyncMapping() throws Exception {
+    setupAsyncMapping();
+    CommandStringBuilder csb = new CommandStringBuilder(DESTROY_MAPPING);
```

Comment:
Multiple regions can be configured to have same AsyncEventQueue; Not sure if jdbc connector allows multiple regions to have the same async queue; if it allows, it will be good to have tests that  make sure the event queue mapping is removed from only the expected region.

Reply:
create jdbc-mapping always creates a unique queue for each region. It is 1-to-1. It does not support sharing a single queue on multiple regions.